docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading logs by using the CLI
```

Comment:
I mentioned in a conversation that this "by using" phrase seems to have been one person's quirk. In the official OpenShift docs, I see "using the CLI" or console. I think that sounds better. Please change here and elsewhere.